src/constants/appConstants.js

```diff
@@ -19,3 +19,5 @@
 */
 
 export const IS_APP_VERSION_V3 = 'isAppVersionV3';
+export const API_REQUEST_TIMEOUT = 10000;
```

Comment:
Is 10 secs enough? Default was 60 secs ifrc

Reply:
It was always 10s, this hasn't changed.